src/pages/minjeong/Login/loginMj.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
+import React, { useState } from 'react';
+import './loginMj.scss';
+import { useNavigate } from 'react-router-dom';
+
+function LoginMj() {
+  const navigate = useNavigate();
+  let [id, setId] = useState('');
+  let [pw, setPw] = useState('');
+  let [isValid, setIsValid] = useState(false);
+
+  function saveUserId(event) {
+    setId(event.target.value);
+    console.log('id :', id);
+  }
+
+  function saveUserPw(event) {
+    setPw(event.target.value);
+    console.log('pw :', pw);
+  }
+
+  const goToMain = () => {
+    navigate('/mainmj');
+  };
+
+  const isActive = id.includes('@') && pw.length >= 5;
+
+  return (
+    <div className="Login">
+      <div className="inputBox">
+        <p className="westagram">Westagram</p>
+        <input
+          className="inputId"
+          type={'text'}
```

Comment:
```suggestion
          type='text'
```
문자열은 중괄호로 안 감싸 주셔도 됩니다!